index.html

```diff
@@ -69,5 +69,167 @@ <h1>
         </h1>
       </div>
     </section>
+    <section class="works">
+    <div class="multi-stories">
+     <div class="grayBg"></div>
+     <div class="txt inter-font">
+        <h2>Multi-Post Stories</h2>
+        <h2>Gain+Glory</h2>
+        <ul class="projectBTNs inter-font">
+            <li> <button>Ruby on rails</button></li>
+            <li><button>css</button></li>
+            <li><button>javascript</button></li>
+            <li><button>html</button></li>
+        </ul>
+        <button class="seeProject inter-font">See Project</button>
+     </div>
+    </div>
+    <div class="multi-stories">
+      <div class="grayBg"></div>
+      <div class="txt inter-font">
+         <h2>Multi-Post Stories</h2>
+         <h2>Gain+Glory</h2>
+         <ul class="projectBTNs inter-font">
+             <li> <button>Ruby on rails</button></li>
+             <li><button>css</button></li>
+             <li><button>javascript</button></li>
+             <li><button>html</button></li>
+         </ul>
+         <button class="seeProject inter-font">See Project</button>
+      </div>
+     </div>
+     <div class="multi-stories">
+      <div class="grayBg"></div>
+      <div class="txt inter-font">
+         <h2>Multi-Post Stories</h2>
+         <h2>Gain+Glory</h2>
+         <ul class="projectBTNs inter-font">
+             <li> <button>Ruby on rails</button></li>
+             <li><button>css</button></li>
+             <li><button>javascript</button></li>
+             <li><button>html</button></li>
+         </ul>
+         <button class="seeProject inter-font">See Project</button>
+      </div>
+     </div>
+     <div class="multi-stories">
+      <div class="grayBg"></div>
+      <div class="txt inter-font">
+         <h2>Multi-Post Stories</h2>
+         <h2>Gain+Glory</h2>
+         <ul class="projectBTNs inter-font">
+             <li> <button>Ruby on rails</button></li>
+             <li><button>css</button></li>
+             <li><button>javascript</button></li>
+             <li><button>html</button></li>
+         </ul>
+         <button class="seeProject inter-font">See Project</button>
+      </div>
+     </div>
+     <div class="multi-stories">
+      <div class="grayBg"></div>
+      <div class="txt inter-font">
+         <h2>Multi-Post Stories</h2>
+         <h2>Gain+Glory</h2>
+         <ul class="projectBTNs inter-font">
+             <li> <button>Ruby on rails</button></li>
+             <li><button>css</button></li>
+             <li><button>javascript</button></li>
+             <li><button>html</button></li>
+         </ul>
+         <button class="seeProject inter-font">See Project</button>
+      </div>
+     </div>
+     <div class="multi-stories">
+      <div class="grayBg"></div>
+      <div class="txt inter-font">
+         <h2>Multi-Post Stories</h2>
```

Comment:
- [ ] As suggested by the previous reviewer, please kindly ensure that your cards are centered on the page like it is in the design.
- [ ] As suggested by the previous reviewer, I also kindly suggest that you stick as much as possible to the design. See the design for reference

| Your Design  | Figma Design |
| ------------- | ------------- |
|<img width="242" alt="image" src="https://user-images.githubusercontent.com/52934128/185574972-fd2230d4-1e0b-4f9e-8e59-cb95a563e589.png">  | <img width="319" alt="image" src="https://user-images.githubusercontent.com/52934128/185575751-1e49165a-406a-43fe-8f09-18483746e7a9.png"> |

Reply:
solved

Reply:
@roshan-bajgain  can you plz check again?